tracing-error/src/backtrace.rs

```diff
@@ -170,8 +169,46 @@ impl fmt::Display for SpanTrace {
     }
 }
 
+struct SpanTraceFrame<'a> {
```

Comment:
Nit/TIOLI: i wouldn't personally call this a "frame", as it's not formatting a stack frame, it's formatting a span. Maybe `DebugSpan`?